code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -457,6 +457,8 @@
 
 	return TRUE
 
+/mob/living/carbon/human/species/monkey/can_be_facehugged(obj/item/clothing/mask/facehugger/F, check_death = TRUE, check_mask = TRUE, provoked = FALSE)
+	return FALSE // monkeys are ignored by facehuggers
```

Comment:
Now that i think about it, sometimes players are monkeys because of admin memery.

Make it return FALSE if there is no mind in the mob (aka no players), yes otherwise

Reply:
What would be the check for that?

Reply:
if(mind)
     return TRUE
return FALSE

should work

Reply:
ahh. turns out i was overcomplicating it. I tried checking for the CKEY and if it returned nothing to make it false

Reply:
solved